packages/components/molecules/f-searchbox/src/components/formElements/tests/FormFullAddressSearchModalOverlay.test.js

```diff
@@ -0,0 +1,146 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+import FormFullAddressSearchModalOverlay from '../FormFullAddressSearchModalOverlay.vue';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+
+const mockState = {
+    isBelowMid: false,
+    isFullAddressSearchEnabled: true,
+    suggestions: ['NG+', 'NG++'],
+    hasInputElevation: false
+};
+
+const mockActions = {
+    setAddress: jest.fn(),
+    getMatchedAreaAddressResults: jest.fn()
+};
+
+const createStore = (state = mockState, actions = mockActions) => new Vuex.Store({
+    modules: {
+        searchbox: {
+            namespaced: true,
+            state,
+            actions
+        }
+    },
+    hasModule: jest.fn(() => true)
+});
+
+describe('`FormFullAddressSearchModalOverlay`', () => {
+    describe('`shouldDisplayModalOverlay`', () => {
+        it('should exist', () => {
+            const wrapper = shallowMount(FormFullAddressSearchModalOverlay, {
```

Comment:
Do you need `// Arrange` and `// Assert` ?

Reply:
Fixed.